go.sum

```diff
@@ -1,3 +1,4 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```

Comment:
i think im doing something wrong..... 

was trying to debug this https://github.com/transcend-io/terraform-provider-transcend/actions/runs/3342880564/jobs/5535551097

Reply:
This changed from the `go generate ./...` command?

Reply:
it required me to install something then i ran `go generate` no other args